lua/xcodebuild/core/xcode.lua

```diff
@@ -523,7 +523,13 @@ function M.install_app_on_simulator(destination, appPath, callback)
       if code ~= 0 then
         notifications.send_error("Could not install app (code: " .. code .. ")")
```

Comment:
I think we shouldn't show this error if we are going to retry, just to avoid confusion that something is wrong :)